renderer/lib/telemetry.js

```diff
@@ -17,6 +17,15 @@ const config = require('../../config')
 var telemetry
 
 function init (state) {
+  // Running the app in development?
+  // Make a dummy telemetry object, don't modify config.json, don't post
+  if (!config.IS_PRODUCTION) {
+    telemetry = createSummary()
```

Comment:
Minor nit: I think this doesn't set the other telemetry properties like `timestamp` which will make it harder to debug those in dev. I'll fix up post-merge.